mmengine/config/config.py

```diff
@@ -232,6 +450,74 @@ def fromstring(cfg_str: str, file_format: str) -> 'Config':
         os.remove(temp_file.name)  # manually delete the temporary file
         return cfg
 
+    @staticmethod
+    def _get_base_modules(codes: list) -> list:
+        """Get base module name from parsed code.
+
+        Args:
+            codes (str): Parsed code of the config file.
+
+        Returns:
+            list: Name of base modules.
+        """
+
+        def _get_base_module_from_if(ifcodes: list) -> list:
+            """Get base module name from if statement in python file.
+
+            Args:
+                ifcodes (list): List of if statement.
+
+            Returns:
+                list: Name of base modules.
+            """
+            base_modules = []
+            for inst in ifcodes:
+                assert isinstance(inst, ast.ImportFrom), (
+                    'Only support import from statement in _base_')
+                assert inst.module is not None, (
+                    'You must import base file by from xxx import yyy')
+                base_modules.append(inst.level * '.' + inst.module)
+            return base_modules
+
+        for idx, inst in enumerate(codes):
```

Comment:
What is the meaning of `inst`?

Reply:
`inst` means the instruction, which is an `ast.Node` object. I just follow the manner of PyTorch Dynamo, which names the bytecode as `inst`